packages/wizard/src/wizard.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  chakra,
+  forwardRef,
+  HTMLChakraProps,
+  omitThemingProps,
+  StylesProvider,
+  ThemingProps,
+  useMultiStyleConfig,
+} from "@chakra-ui/system"
+import { cx } from "@chakra-ui/utils"
+import * as React from "react"
+
+export interface WizardProps extends HTMLChakraProps<"div">, ThemingProps {
+  activeStep: number
+  orientation?: "vertical" | "horizontal"
+  labelOrientation?: "vertical" | "horizontal"
```

Comment:
Is there a situation where the overall orientation is different than the label orientation? I could not find an example in the storybook so I would drop support for label orientation in this PR to keep things simple